src/content/h1/info/readme.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hard-coded Data
+keywords:
+  - color
+  - data
+  - engine
+  - hard-coded
+  - info
+thanks:
+  - to: MosesOfEgypt
+    for: Armor color values
+  - to: Lavadeeto
+    for: Armor color testing and picture
+---
+There are lots of hard-coded values in Halo's engine, such as the multiplayer
+armor color. These values do not typically appear anywhere in tag data, and
+cannot be directly changed or controlled.
+
+# Multiplayer Armor Colors
```

Comment:
Minor, but the page title defined in YAML and any headings in the markdown should be in "Sentence case" like Wikipedia rather than "Title Case".